vue-app/src/views/AboutLayer2.vue

```diff
@@ -57,33 +64,203 @@
-      >.
-    </p>
-
-    <h2>What you'll need</h2>
-    <ul>
-      <li>A wallet that supports Arbitrum</li>
-      <li>Funds on Abritrum</li>
-    </ul>
-    <h3>How to find wallet that supports Arbitrum</h3>
-    <p>TODO</p>
-    <h3>How to get funds on Arbitrum</h3>
-    <p>TODO</p>
+      <h2>What you'll need</h2>
+      <ul>
+        <li>A wallet that supports {{ chain.label }}</li>
+        <li>Funds on {{ chain.label }}</li>
+      </ul>
+      <h3>
+        💼 How to find wallet that supports {{ chain.label }}
+        <img
+          v-tooltip="{
+            content:
+              'Wallet resources are provided as a convenience and do not represent endorsement of any of the projects or services therein. Always DYOR.',
+            trigger: 'hover click',
+          }"
+          width="16px"
+          src="@/assets/info.svg"
+        />
+      </h3>
+      <ul>
+        <li>
+          Visit the official
+          <links to="https://portal.arbitrum.one/">
+            {{ chain.label }} portal
+          </links>
+          and filter by "Wallets" to view some of the wallets that currently
+          support the {{ chain.label }} network.
+        </li>
+        <li>
+          Double-check that any wallet you consider
+          <links to="https://registry.walletconnect.org/wallets">
+            also supports WalletConnect
+          </links>
+          to ensure you're able to connect to the app.
+        </li>
+      </ul>
+      <h3>💰 How to get funds on {{ chain.label }}</h3>
+      <p>
+        <links :to="chain.bridge" :hideArrow="true">
+          <button class="btn-action">Official {{ chain.label }} Bridge</button>
+        </links>
+      </p>
+      <p>
+        Follow the steps below, or use the
+        <links to="https://arbitrum.io/bridge-tutorial/">
+          official tutorial
+        </links>
+        as a guide at any time.
+      </p>
+      <ol>
+        <li>Click above to go to the official {{ chain.label }} bridge</li>
+        <li>
+          Connect your {{ chain.label }} supporting wallet using
+          <strong>Mainnet</strong>
+        </li>
+        <li>
+          Select currency (some ETH first for gas, and some
+          {{ nativeToken.symbol }} for contributing)
+          <p>
+            For {{ nativeToken.symbol }}, click "Token" menu, search for
+            {{ nativeToken.symbol }}
+            and select token.
+          </p>
+        </li>
+        <li>Enter amount and click "Deposit"</li>
+        <li>Confirm on your wallet</li>
+      </ol>
+      <p>
+        Once you have bridged your {{ nativeToken.symbol }} to
+        {{ chain.label }}, you may want to add the
+        <links :to="blockExplorerUrl">token</links> to your wallet e.g. in
+        MetaMask.
+      </p>
+      <button
+        v-if="currentUser && isMetaMask"
+        class="btn-secondary"
+        @click="addTokenToWallet"
+      >
+        Add {{ chain.label }} {{ nativeToken.symbol }} to MetaMask
```

Comment:
Oh yeah! would be nice to have an open source package (as a plugin) for this buttons :)